models/yield_yak/yield_yak_schema.yml

```
@@ -248,4 +248,118 @@ models:
         description: "UTC time until which the user held this balance of the contract_address YRT"
       - &yrt_balance
         name: yrt_balance
-        description: "Balance of YRT tokens held by the user_address in the time window specified"
\ No newline at end of file
+        description: "Balance of YRT tokens held by the user_address in the time window specified"
+  - name: yield_yak_yak_swaps
```

You can add a blank line between model definition:
```suggestion
        description: "Balance of YRT tokens held by the user_address in the time window specified"

  - name: yield_yak_yak_swaps
```

---

Okay sure, will make this change, plus will add blank lines between other models and will also add the explicit `materialized = 'view'` line on some of the other relevant models that I've put in spellbook a while ago for this project :)